### PR TITLE
Create sample extension without Shell side navigation

### PR DESCRIPTION
**Pull request by kistll**

### CODE REVIEW

File: samples/without-shell-navigation/appconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "1.0.0", 
+  "icon": "https://raw.githubusercontent.com/SAP-samples/fsm-extension-sample/master/samples/without-shell-navigation/icon.png",
+  "lang": "en",
+  "translations": [
+    {
+      "lang": "de",
+      "name": "Erweiterung ohne Shell Navigation",
+      "description": "Beispielerweiterung innerhalb der Shell ohne Shell Navigation"
+    }
+  ],
```
Comment (sebastienbarbier) [inline samples/without-shell-navigation/appconfig.json:14]: Could we add english 😎

File: samples/without-shell-navigation/helpers.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  document.getElementById('first-level-2-second-level-1-text').innerHTML = I18N.FIRST_LEVEL_2_SECOND_LEVEL_1_TEXT;
+  document.getElementById('first-level-2-second-level-1-title').innerHTML = I18N.FIRST_LEVEL_2_SECOND_LEVEL_1_TITLE;
+  document.getElementById('side-first-level-1-title').innerHTML = I18N.SIDE_FIRST_LEVEL_1_TITLE;
+  document.getElementById('side-first-level-2-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_TITLE;
+  document.getElementById('side-first-level-2-side-second-level-1-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_1_TITLE;
+  document.getElementById('side-first-level-2-side-second-level-2-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_2_TITLE;
+  document.getElementById('side-first-level-2-side-second-level-2-side-third-level-1-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_2_SIDE_THIRD_LEVEL_1_TITLE;
+  document.getElementById('first-level-2-second-level-2-third-level-1-text').innerHTML = I18N.SECOND_LEVEL_2_FIRST_LEVEL_2_THIRD_LEVEL_1_TEXT;
+  document.getElementById('first-level-2-second-level-2-third-level-1-title').innerHTML = I18N.SECOND_LEVEL_2_FIRST_LEVEL_2_THIRD_LEVEL_1_TITLE;
```
Comment (sebastienbarbier) [inline samples/without-shell-navigation/helpers.js:169]: What about a dictionary with key: value then a for loop using the `document.getElementById(key).innerHTML = value` ?

File: samples/without-shell-navigation/index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Extension with Shell navigation</title>
+    <link href="https://unpkg.com/fundamental-styles" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+  </head>
+  <body style="margin: 0;">
+    <div>     
+      <div>
```
Comment (sebastienbarbier) [inline samples/without-shell-navigation/index.html:11]: Why so many `<div>` ?  i would simplify to make html easier to read.

**Review by sebastienbarbier — CHANGES_REQUESTED** [review]

File: samples/without-shell-navigation/appconfig.json (hunk 1 above)
Comment (kistll) [reply samples/without-shell-navigation/appconfig.json:14]: English is already available. The default language is not part of the translations array. It is provided by the attributes lang, name and description.

**Review by sebastienbarbier — APPROVED** [review]

**Pull request merged**